contracts/HybridCustody.cdc

```diff
@@ -55,6 +55,7 @@ pub contract HybridCustody {
         pub fun isChildOf(_ addr: Address): Bool
         pub fun getParentsAddresses(): [Address]
         pub fun borrowAccount(): &AuthAccount?
+        pub fun getOwner(): Address?
```

Comment:
Perhaps less important, but I think we should also add `getPendingOwner()` here. What do you think?